### DISCUSSION TITLE
21차 모임 (2024-03-26) 그룹 토의 주제

### DISCUSSION THREAD

**Discussion by haksungjang** [OpenChain-KWG/meeting#5] [General]
안녕하세요. 

21차 ('24년 1분기) 정기 모임이 3월26일(화)에 예정되어 있습니다. : https://openchain-project.github.io/OpenChain-KWG/meeting/21th/
이 때 그룹 토의를 진행할 예정인데요, 함께 논의하시고 싶은 주제가 있다면 comment로 남겨주시기 바랍니다. :) 

감사합니다.

**Comment by yoobato** [comment]
오픈소스 컴플라이언스 관련 업무를 하다보면, 사내 직원의 질문에 판단 기준이 서지 않는 경우가 많아 인터넷 검색(특히 공개SW 포털)을 해서 안내드리는 경우가 종종 있는데요.
- 최근에도 GPL 2.0 오픈소스 바이너리를 회사 서버에 올려 배포 시 어떤 의무사항을 준수해야하는 문의가 있었는데, 이걸 미러링 저장소의 개념으로 봐야할지 오픈소스 재배포로 봐야할지 애매하더라구요 😭

혹시, 국내 각 기업 오픈소스 담당자들끼리 자유롭게 커뮤니케이션할 수 있는 채널을 만들면 어떨까요? (예. Slack)

**Reply by haksungjang** [reply]
넵, 의견 감사합니다~ 이번 모임에서 함께 논의해보면 좋겠습니다. 👍
